petastorm/codecs.py

```diff
@@ -27,6 +29,7 @@
 
```

Comment:
Unfortunately, the sphinx build is unable to deal with `cv2` import without blowing memory limitations (free service... apparently the boxes only get 1 GB RAM).  So, `codec` won't get processed.

One alternative is if `import cv2` happens only within `CompressedImageCodec.__init__`?! :D

Reply:
Another option is to just not import it when doing a read the docs? https://docs.readthedocs.io/en/latest/faq.html#how-do-i-change-behavior-for-read-the-docs

Reply:
So, either way, the `codecs.py` would simply to skipped. It doesn't cause the entire autodoc build process to fail.